tests/e2e/main.go

```diff
@@ -192,6 +192,12 @@ var stepChoices = map[string]StepChoice{
 		description: "test partial set security for an Opt-In chain that has a validator denylisted",
 		testConfig:  DefaultTestCfg,
 	},
+	"active-set-changes": {
+		name:        "active-set-changes",
+		steps:       stepsActiveSetChanges(),
+		description: "",
```

Comment:
Maybe add a description. It might be nice to also link to the post-mortem here for posterity?